libs/openFrameworksCompiled/project/tvOS/tvOS+OFLib.xcodeproj/project.pbxproj

```diff
@@ -1245,16 +1220,20 @@
 		844639661BC343E000F24926 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				CONFIGURATION_TEMP_DIR = "$(SRCROOT)/../../lib/tvos/build/debug/";
+				BUILD_DIR = "~/Library/Developer/Xcode/DerivedData";
```

Comment:
Should we be hard coding the build dir? 
I think before it respected the user setting right?   

<img width="526" alt="image" src="https://github.com/user-attachments/assets/c26ed900-e6da-412f-9b8e-ebf3df6c143a" />